DriverDataUI/src/main/java/com/drivequant/drivekit/ui/tripdetail/viewmodel/TripDetailViewModel.kt

```diff
@@ -303,8 +303,6 @@ internal class TripDetailViewModel(
     private fun getAdviceByMapItem(mapItem: DKMapItem): TripAdvice? {
         return trip?.let {
```

Comment:
On devrait pouvoir simplifier le code en faisant : 

`  private fun getAdviceByMapItem(mapItem: DKMapItem): TripAdvice? = trip?.let {
            mapItem.getAdvice(it)
        }
    }`